native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -61,10 +74,19 @@ const base = StyleSheet.create({
   },
 
   // Applied to the Text element.
-  text: {
-    backgroundColor: 'transparent',
-    color: tokens.colorWhite,
-  },
+  text: Platform.select({
+    ios: () => ({
+      backgroundColor: 'transparent',
+      color: tokens.colorWhite,
+
+    }),
+    android: () => ({
+      backgroundColor: 'transparent',
+      color: tokens.colorWhite,
+      // padding: tokens.spacingSm,
```

Comment:
Should this be removed, as it's commented out?

Reply:
yes I saw that, and I've fixed it 😄